checkQC/qc_data_utils.py

```diff
@@ -196,4 +196,8 @@ def bclconvert_test_runfolder(qc_data, runfolder_path):
                 },
             },
         }
+    else:
+        raise Exception("Excpected flowcell_id value as 'HMTFYDRXX' only for "
+                        f"this fuction but got {flowcell_id}"
```

Comment:
Oh, there is a typo in the exception. Also, I thought the exception could be rephrased to something like this. 

```suggestion
        raise Exception("This function is only compatible with the run with flowcell_id: 'HMTFYDRXX', "
                                  f"the supplied runfolder has flowcell_id: {flowcell_id}"
```